scrapyd_k8s/launcher/k8s.py

```diff
@@ -121,8 +135,65 @@ def cancel(self, project, job_id, signal):
             )
         return prevstate
 
-    def enable_joblogs(self, config):
-        joblogs_init(config)
+    def enable_joblogs(self, config, resource_watcher):
+        joblogs_init(config, resource_watcher)
+
+    def unsuspend_job(self, job_id: str):
```

Comment:
is "unsuspend" the k8s terminology? I would also imagine resume, or start or so.

Reply:
The native term for k8s is "suspend", this is the particular field that we set to True/False when we create a job, that if why "unsuspend" seems a good fit and quite descriptive for Kubernetes.